sdk/eventgrid/test-resources.json

```diff
@@ -92,40 +101,97 @@
+        "principalId": "[parameters('testApplicationOid')]"
+      }
+    },
+    {
+      "type": "Microsoft.EventGrid/namespaces",
+      "apiVersion": "2024-06-01-preview",
+      "name": "[variables('egNamespaceName')]",
+      "location": "eastus",
+      "sku": {
+        "name": "Standard",
+        "capacity": 1
+      },
+      "properties": {}
+    },
+    {
+      "type": "Microsoft.EventGrid/namespaces/topics",
+      "apiVersion": "2024-06-01-preview",
+      "name": "[concat(variables('egNamespaceName'), '/testtopic1')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.EventGrid/namespaces', variables('egNamespaceName'))]"
+      ],
+      "properties": {
+        "publisherType": "Custom",
+        "inputSchema": "CloudEventSchemaV1_0",
+        "eventRetentionInDays": 1
+      }
+    },
+    {
+      "type": "Microsoft.EventGrid/namespaces/topics/eventSubscriptions",
+      "apiVersion": "2024-06-01-preview",
+      "name": "[concat(variables('egNamespaceName'), '/testtopic1/testsubscription1')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.EventGrid/namespaces/topics', variables('egNamespaceName'), 'testtopic1')]",
+        "[resourceId('Microsoft.EventGrid/namespaces', variables('egNamespaceName'))]"
+      ],
+      "properties": {
+        "deliveryConfiguration": {
+          "deliveryMode": "Queue",
+          "queue": {
+            "receiveLockDurationInSeconds": 60,
+            "maxDeliveryCount": 10,
+            "eventTimeToLive": "P1D"
+          }
+        },
+        "eventDeliverySchema": "CloudEventSchemaV1_0",
+        "filtersConfiguration": {
+          "includedEventTypes": []
+        }
+      }
     }
   ],
   "outputs": {
     "EVENT_GRID_EVENT_GRID_SCHEMA_ENDPOINT": {
       "type": "string",
       "value": "[reference(variables('eventGridTopicName')).endpoint]"
     },
-    "EVENT_GRID_EVENT_GRID_SCHEMA_API_KEY": {
-      "type": "string",
-      "value": "[listKeys(variables('eventGridTopicName'), '2020-06-01').key1]"
-    },
     "EVENT_GRID_CLOUD_EVENT_SCHEMA_ENDPOINT": {
       "type": "string",
       "value": "[reference(variables('cloudEventTopicName')).endpoint]"
     },
-    "EVENT_GRID_CLOUD_EVENT_SCHEMA_API_KEY": {
-      "type": "string",
-      "value": "[listKeys(variables('cloudEventTopicName'), '2020-06-01').key1]"
-    },
     "EVENT_GRID_CUSTOM_SCHEMA_ENDPOINT": {
       "type": "string",
       "value": "[reference(variables('customEventTopicName')).endpoint]"
     },
-    "EVENT_GRID_CUSTOM_SCHEMA_API_KEY": {
-      "type": "string",
-      "value": "[listKeys(variables('customEventTopicName'), '2020-06-01').key1]"
-    },
     "SERVICE_BUS_CONNECTION_STRING": {
       "type": "string",
       "value": "[listKeys(resourceId('Microsoft.ServiceBus/namespaces/AuthorizationRules',variables('serviceBusNamespaceName'),'RootManageSharedAccessKey'),'2017-04-01').primaryConnectionString]"
     },
```

Comment:
```suggestion
```
The connection string is also considered a secret and should be removed from the output too.